homeassistant/components/light/__init__.py

```diff
@@ -107,6 +110,7 @@
     ATTR_WHITE_VALUE: vol.All(vol.Coerce(int), vol.Range(min=0, max=255)),
     ATTR_FLASH: vol.In([FLASH_SHORT, FLASH_LONG]),
     ATTR_EFFECT: cv.string,
+    vol.Optional(ATTR_POWER_ON, default=True): cv.boolean,
```

Comment:
It's a lifx only service so I don't really care. But I feel like it would make sense to keep them in sync with the rest of Hass. So rather not use the app units.